app/networkapi/urls.py

```diff
@@ -29,9 +29,11 @@
     url(r'^api/people/', include('networkapi.people.urls')),
     url(r'^api/news/', include('networkapi.news.urls')),
     url(r'^$', mezzanine.pages.views.page, {'slug': '/'}, name='home'),
+    url(r'^', include('pulseapi.urls')),
```

Comment:
So looking at this, I feel like long term these urls should go under the `/api` prefix, for consistencies sake.

what if we expose the views on the old routes and on `/api` to give us a window to update things that depend on those routes.